modules/indexer/issues/indexer.go

```diff
@@ -268,7 +281,7 @@ func DeleteRepoIssueIndexer(repo *models.Repository) {
 // SearchIssuesByKeyword search issue ids by keywords and repo id
 func SearchIssuesByKeyword(repoIDs []int64, keyword string) ([]int64, error) {
 	var issueIDs []int64
-	res, err := holder.get().Search(keyword, repoIDs, 1000, 0)
+	res, err := holder.get().Search(keyword, repoIDs, 50, 0)
```

Comment:
Is it possible that the first 50 issues are not accesible for the querying user?

Reply:
@guillep2k Currently we have no global issue search, so this should be the reposibility of the invoker to ensure user have permission to visit repoIDs' issues. I will add a warnning on the function to keep the work for other PRs.